source/service.go

```diff
@@ -70,7 +75,7 @@ func (sc *serviceSource) Endpoints() ([]*endpoint.Endpoint, error) {
 }
 
 // endpointsFromService extracts the endpoints from a service object
-func endpointsFromService(svc *v1.Service) []*endpoint.Endpoint {
+func endpointsFromService(svc *v1.Service, fqdntemplate string) []*endpoint.Endpoint {
```

Comment:
possible makes sense to have this as a method on ServiceSource object, so you can reference the tmpl object